pkg/sdk/go/things_test.go

```diff
@@ -36,7 +35,7 @@ const (
 var (
 	metadata   = map[string]interface{}{"meta": "data"}
 	metadata2  = map[string]interface{}{"meta": "data2"}
-	thing      = sdk.Thing{ID: "1", Name: "test_device", Metadata: metadata}
+	thing      = sdk.Thing{ID: "001", Name: "test_device", Metadata: metadata}
```

Comment:
Try to use UUID

Reply:
Point is that here we mock id so we can test "order by id" functionalities.
That is the reason why I have change mock IDs to use three chars instead one, because ordering with just one char, like it is now, is `1`, `10`, `100`, `2`, `20`...
With this change order would be `001`, `002`, `003`... and you can write tests that check order by id functionality.